trl/trainer/grpo_trainer.py

```diff
@@ -275,6 +276,35 @@ def data_collator(features):  # No data collation is needed in GRPO
             optimizers=optimizers,
         )
 
+        if self.use_vllm:
+            if not is_vllm_available():
+                raise ImportError(
+                    "vLLM is not available and `use_vllm` is set to True. Please install vLLM with "
+                    "`pip install vllm` to use it."
+                )
+
+            if self.accelerator.is_main_process:  # for demo, run only in the main process
```

Comment:
I don't understand what is meant by "demo" here. Remove?
```suggestion
            if self.accelerator.is_main_process: 
```

Reply:
right, wasn't supposed to be here